telegram/_chatbackground.py

```diff
@@ -0,0 +1,494 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2024
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains objects related to chat backgrounds."""
+from typing import TYPE_CHECKING, Dict, Final, Optional, Sequence, Type
+
+from telegram import constants
+from telegram._files.document import Document
+from telegram._telegramobject import TelegramObject
+from telegram._utils import enum
+from telegram._utils.argumentparsing import parse_sequence_arg
+from telegram._utils.types import JSONDict
+
+if TYPE_CHECKING:
+    from telegram import Bot
+
+
+class BackgroundFill(TelegramObject):
+    """Base class for Telegram BackgroundFill Objects. It can be one of:
```

Comment:
I would prefer to stick to the documentation of Telegram as closely as possible

Reply:
hmm. but currently we're using this as a template for almost all telegram base classes
```python

class MessageOrigin(TelegramObject):
    """
    Base class for telegram MessageOrigin object, it can be one of:
    ..."""

class MaybeInaccessibleMessage(TelegramObject):
    """Base class for Telegram Message Objects.
    ..."""

class ReactionType(TelegramObject):
    """Base class for Telegram ReactionType Objects.
    ..."""

class MessageOrigin(TelegramObject):
    """
    Base class for telegram MessageOrigin object, it can be one of:
    ..."""

class ChatBoostSource(TelegramObject):
    """
    Base class for Telegram ChatBoostSource objects. It can be one of:
    ..."""
```

Reply:
mh, keep it for the base class, then, but at least for the subclasses I'd like to mimic TG

Reply:
In that case, already done ^^